processing/src/main/java/io/druid/guice/PropertiesModule.java

```diff
@@ -86,6 +87,16 @@ public void configure(Binder binder)
       }
     }
 
+    log.info(
+        "Loaded properties into JVM with processors[%,d], memory[%,d].",
+        Runtime.getRuntime().availableProcessors(),
+        Runtime.getRuntime().totalMemory()
+    );
+
+    for (String propertyName : Ordering.natural().sortedCopy(props.stringPropertyNames())) {
+      log.info("* %s: %s", propertyName, props.getProperty(propertyName));
```

Comment:
Then we should remove support for the way that puts plain text credentials into the log by default.